analytics/terraform/spark-eks-ipv6/examples/karpenter/ebs-storage-dynamic-pvc/ebs-storage-dynamic-pvc.yaml

```diff
@@ -0,0 +1,158 @@
+# Pre-requisite before running this job
+# 1/ Open taxi-trip-execute.sh and update <S3_BUCKET> and <REGION>
+# 2/ Replace <S3_BUCKET> with your S3 bucket created by this blueprint(Check Terraform outputs)
+# 3/ execute taxi-trip-execute.sh
+
+# This example supports the following features
+  # Support shuffle data recovery on the reused PVCs (SPARK-35593)
+  # Support driver-owned on-demand PVC (SPARK-35182)
+# WARNING: spark-operator cluster role is missing a 'persistenvolumeclaims' permission. Ensure you add this permission to spark-operator cluster role
+
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: "taxi-trip"
+  namespace: spark-team-a
+  labels:
+    app: "taxi-trip"
+    applicationId: "taxi-trip-ebs"
+    queue: root.test
+spec:
+#  To create Ingress object for Spark driver.
+#  Ensure Spark Operator Helm Chart deployed with Ingress enabled to use this feature
+#  sparkUIOptions:
+#    servicePort: 4040
+#    servicePortName: taxi-trip-ui-svc
+#    serviceType: ClusterIP
+#    ingressAnnotations:
+#      kubernetes.io/ingress.class: nginx
+#      nginx.ingress.kubernetes.io/use-regex: "true"
+  type: Python
+  sparkVersion: "3.2.1"
+  pythonVersion: "3"
+  mode: cluster
+  image: "public.ecr.aws/r1l5w1y9/spark-operator:3.2.1-hadoop-3.3.1-java-11-scala-2.12-python-3.8-latest"
```

Comment:
We need to replace this with the Dockerfile image provided above. Please create a multi-arch image using the Dockerfile, push it to the DoEKS Public ECR, and update this value accordingly.